web-api-security/src/main/java/com/epam/learning/backendservices/security/model/Person.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.learning.backendservices.security.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "persons")
+public class Person {
+    @Id
+    @Column(name="personId")
+    private Long id;
+
+    @Column(name="personName")
```

Comment:
I think that person prefix is redundant since the db table is called persons. Optional: rename fields.

Reply:
Done